src/views/Home.vue

```diff
@@ -14,136 +11,144 @@
       </div>
     </section>
 
-    <!-- About Section -->
-    <section id="about" class="about-section">
-      <span class = about-content>
+    <!-- about-game Section -->
+    <section id="about-game" class="about-game-section">
+      <span class=about-game-content>
         <h2 ref="popOutText" class="pop-out-text">What is KeyClash</h2>
       </span>
-  
-      <!-- Additional content for the about section can be added here -->
+
+      <!-- Additional content for the about-game section can be added here -->
     </section>
-    <section id="about-us" class="about-us-section">
-      <span class = about-us-content>
+    <section id="meet-team" class="meet-team-section">
+      <span class=meet-team-content>
         <h2 ref="popOutText" class="pop-out-text">Here we are!</h2>
       </span>
-  
+
     </section>
   </div>
 </template>
 
-<script>
+<script setup>
 import { useRouter } from 'vue-router'
 import { ref, onMounted, onBeforeUnmount } from 'vue'
-export default {
-  name: 'Home',
-  setup() {
-    const popOutText = ref(null)
-    const router = useRouter()
-
-    // Function to handle scroll event for pop-out text animation
-    const handleScroll = () => {
-      const popOutTexts = document.querySelectorAll('.pop-out-text')
-      
-      popOutTexts.forEach((element) => {
-        const position = element.getBoundingClientRect().top
-        const screenPosition = window.innerHeight / 1.1
-
-        if (position < screenPosition) {
-          element.classList.add('visible')
-        } else {
-          element.classList.remove('visible')
-        }
-      })
-    }
-    // Lifecycle hooks
-    onMounted(() => {
-      window.addEventListener('scroll', handleScroll)
-    })
-
-    onBeforeUnmount(() => {
-      window.removeEventListener('scroll', handleScroll)
-    })
-    const startGame = () => {
-      router.push('/game')
-    }
-    return {
-      popOutText,
-      handleScroll,
-      startGame
+const popOutText = ref(null)
+const router = useRouter()
+
+// Function to handle scroll event for pop-out text animation
+const handleScroll = () => {
+  const popOutTexts = document.querySelectorAll('.pop-out-text')
+
+  popOutTexts.forEach((element) => {
+    const position = element.getBoundingClientRect().top
+    const screenPosition = window.innerHeight / 1.1
```

Comment:
Why divide by 1.1? Perhaps we could name this as a variable to get rid of the magic number.